monitoring/local_monitoring/ReadMe.md

```diff
@@ -0,0 +1,68 @@
+# Local Monitoring with Kruize-Autotune
```

Comment:
You can remove autotune and just Kruize would do.

Reply:
done!